### ISSUE TITLE
NTR: Lower cranial nerve dysfunction

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4155]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Lower cranial nerve dysfunction

### 2. Synonyms
'Abnormal lower cranial nerve function'

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
A functional abnormality affecting the lower cranial nerves, which include the paired 9th (glossopharyngeal), 10th (vagal), 11th (accessory) and 12th (hypoglossal) cranial nerves.

PMID: 26167022

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP_0031910 'Abnormal cranial nerve physiology'
